examples/nidaqmx_analog_input/pyproject.toml

```diff
@@ -13,11 +13,10 @@ click = ">=7.1.2"
 [tool.poetry.group.dev.dependencies]
 mypy = ">=1.0"
 grpc-stubs = "^1.53"
-# Uncomment to use prerelease dependencies (requires poetry>=1.2).
+# Uncomment to use prerelease dependencies.
```

Comment:
Does this refer to line 19 below? May want to move it directly above that line since it appears to refer to nidaqmx right now.

Reply:
Or maybe the nidaqmx line is supposed to be commented too?

Reply:
It refers to both. nidaqmx will be commented out once 0.8.0-dev0 is released.